tests/mir-opt/coroutine_drop_cleanup.main-{closure#0}.coroutine_drop.0.panic-abort.mir

```diff
@@ -1,25 +1,4 @@
 // MIR for `main::{closure#0}` 0 coroutine_drop
-/* coroutine_layout = CoroutineLayout {
```

Comment:
(Unsurprisingly, coroutine layout is not present in the drop shim body anymore. This does affect the MIR dump, but does not affect the drop's codegen.)